index.html

```diff
@@ -0,0 +1,291 @@
+                />
+              </div>
+              <span>Lecture</span>
+              <p>
+                Listen to the speakers from various countries about the messages
+                of sharing and opening
+              </p>
+            </div>
+            <div class="sectiontwo__container__items__item">
+              <div class="sectiontwo__container__items__item__responsive">
+                <img
+                  src="static/imgs/forum.svg"
+                  alt="forum"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>CC Exhibition</span>
+              <p>
+                Appreciate various creations applying CC license of participants
+                from this community
+              </p>
+            </div>
+            <div class="sectiontwo__container__items__item">
+              <div class="sectiontwo__container__items__item__responsive">
+                <img
+                  src="static/imgs/ccIgnite.svg"
+                  alt="ccIgnite"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>Forum</span>
+              <p>
+                Have the time to share your thoughts and opinionswith experts
+                for each topic
+              </p>
+            </div>
+            <div class="sectiontwo__container__items__item">
+              <div class="sectiontwo__container__items__item__responsive">
+                <img
+                  src="static/imgs/workshop.svg"
+                  alt="workshop"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>Workshop</span>
+              <p>
+                Denotaing and small community using open sources and license
+                than sitting
+              </p>
+            </div>
+            <div class="sectiontwo__container__items__item">
+              <div class="sectiontwo__container__items__item__responsive">
+                <img
+                  src="static/imgs/workshop.svg"
+                  alt="workshop"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>CC Ignite </span>
+              <p>
+                Get opportunity to network the CC affiliates around the world,
+                also after conference
+              </p>
+            </div>
+          </div>
+          <a class="sectiontwo__container__title display" href="#"
+            >SEE THE WHOLE PROGRAM</a
+          >
+          <a class="sectiontwo__container__title" href="#" id="second">
+            Join unicef's annual human support</a
+          >
+        </div>
+      </section>
```

Comment:
- [ ] Kindly note that it is a requirement to use the same color theme as the ones provided in the Behance design. Currently, the background color of this section does not match the one in the Behance design.
- [ ] Also, it is a requirement that the layout of things be the same as in the Behance design. The section cards are required to be aligned in one line and not in two. Kindly check the padding so that they are the same as in the Behance design.

**This is how this section is looking right now:**

![image](https://user-images.githubusercontent.com/86318284/197012859-a8e8dfec-6648-4096-856a-efdd79b6229b.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/197012968-c0f4fa37-3e11-4c5c-ab58-207330c45e2c.png)